plugins/quetz_conda_suggest/README.md

```diff
@@ -0,0 +1,12 @@
+# quetz_conda_suggest plugin
+
+This is a plugin to use with the [quetz](https://github.com/mamba-org/quetz) package server.
```

Comment:
can you add a line to say that this produces files that can be used with conda-suggest and a link to conda-suggest?

Reply:
Updating it in a second.